webapp/graphite/carbonlink.py

```diff
@@ -111,7 +123,7 @@ def send_request(self, request):
     result = {}
     result.setdefault('datapoints', [])
 
-    if metric.startswith(settings.CARBON_METRIC_PREFIX):
+    if metric.startswith(settings.CARBON_METRIC_PREFIX) and (request['type'] not in ['get-storageschema', 'cache-query-precheck']):
```

Comment:
something more semantic here would be great:

```
if metric.startswith(settings.CARBON_METRIC_PREFIX) and _is_all_request(request):
```
the _is_all_request is designed to make it clear just by reading the code what the intent is. In this case, it's not immediately clear under what situations I should or shouldn't send add to this list. 
https://refactoring.com/catalog/extractMethod.html